servicenow/resources/resource_scripted_rest_resource.go

```diff
@@ -0,0 +1,228 @@
+}
+
+func resourceFromScriptedRestResource(data *schema.ResourceData, scriptedRestResource *client.ScriptedRestResource) {
+	data.SetId(scriptedRestResource.ID)
+	data.Set(scriptedRestResourceName, scriptedRestResource.Name)
+	data.Set(scriptedRestResourceActive, scriptedRestResource.Active)
+	data.Set(scriptedRestResourceEnforceACL, scriptedRestResource.EnforceACL)
+	data.Set(scriptedRestResourceRequiresACLAuthorization, scriptedRestResource.RequiresACLAuthorization)
+	data.Set(scriptedRestResourceRequiresAuthentication, scriptedRestResource.RequiresAuthentication)
+	data.Set(scriptedRestResourceRequiresSNCInternalRole, scriptedRestResource.RequiresSNCInternalRole)
+	data.Set(scriptedRestResourceProduces, scriptedRestResource.Produces)
+	data.Set(scriptedRestResourceProducesCustomized, scriptedRestResource.ProducesCustomized)
+	data.Set(scriptedRestResourceShortDescription, scriptedRestResource.ShortDescription)
+	data.Set(scriptedRestResourceOperationScript, scriptedRestResource.OperationScript)
+	data.Set(scriptedRestResourceRelativePath, scriptedRestResource.RelativePath)
+	data.Set(scriptedRestResourceRequestExample, scriptedRestResource.RequestExample)
+	data.Set(scriptedRestResourceHTTPMethod, scriptedRestResource.HTTPMethod)
+	data.Set(scriptedRestResourceConsumes, scriptedRestResource.Consumes)
+	data.Set(scriptedRestResourceConsumesCustomized, scriptedRestResource.ConsumesCustomized)
+	data.Set(scriptedRestResourceOperationURI, scriptedRestResource.OperationURI)
+	data.Set(scriptedRestResourceWebServiceDefinition, scriptedRestResource.WebServiceDefinition)
+	data.Set(scriptedRestResourceWebServiceVersion, scriptedRestResource.WebServiceVersion)
+	data.Set(commonProtectionPolicy, scriptedRestResource.ProtectionPolicy)
+	data.Set(commonScope, scriptedRestResource.Scope)
+
+}
+
+func resourceToScriptedRestResource(data *schema.ResourceData) *client.ScriptedRestResource {
+	scriptedRestResource := client.ScriptedRestResource{
+		Name:                     data.Get(scriptedRestResourceName).(string),
+		Active:                   data.Get(scriptedRestResourceActive).(bool),
+		EnforceACL:               data.Get(scriptedRestResourceEnforceACL).(string),
+		RequiresACLAuthorization: data.Get(scriptedRestResourceRequiresACLAuthorization).(bool),
+		RequiresAuthentication:   data.Get(scriptedRestResourceRequiresAuthentication).(bool),
+		RequiresSNCInternalRole:  data.Get(scriptedRestResourceRequiresSNCInternalRole).(bool),
+		ShortDescription:         data.Get(scriptedRestResourceShortDescription).(string),
+		OperationScript:          data.Get(scriptedRestResourceOperationScript).(string),
+		RelativePath:             data.Get(scriptedRestResourceRelativePath).(string),
+		RequestExample:           data.Get(scriptedRestResourceRequestExample).(string),
+		HTTPMethod:               data.Get(scriptedRestResourceHTTPMethod).(string),
+		OperationURI:             data.Get(scriptedRestResourceOperationURI).(string),
```

Comment:
same here, you can remove OperationURI since it's computed and set Customized values to true directly